app/models/user.rb

```diff
@@ -1,8 +1,10 @@
 class User < ApplicationRecord
```

Comment:
- [ ] When I make an API request to register a new user (to /users endpoint), it successfully creates a user but returns an UnknownFormat error (Which means it can't return a JSON response ).